### ISSUE TITLE
[Feature] Add support for running eslint within a subdirectory 

### ISSUE THREAD

**Issue by jackton1** [tj-actions/eslint-changed-files#367]
### Is this feature missing in the latest version?

- [X] I'm using the latest release

### Is your feature request related to a problem? Please describe.

I’ll like to run eslint from a sub directory as opposed to the root folder

### Describe the solution you'd like?

Ability to specify a `path`

```yml
with:
  path: dir1

```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jackton1** [comment]
For now this might be complicated to implement since this would required changing how changed files are detected. i.e you get a list of changed files from the root of the project but can’t pass them directly to eslint running from a subdirectory.
